src/util.ts

```diff
@@ -1,9 +1,17 @@
 import { AxiosError } from 'axios';
 
 export function serialize(obj: unknown): object {
+  if (obj && typeof obj === 'object') {
+    return Object.getOwnPropertyNames(obj).reduce((map, key) => {
+      // eslint-disable-next-line no-param-reassign
+      map[key] = obj[key];
+      return map;
+    }, {});
+  }
+
   return JSON.parse(JSON.stringify(obj, Object.getOwnPropertyNames(obj)));
```

Comment:
We can probably remove this now (i.e. if it's not an object there's probably no need to serialize it)

Reply:
And just return the input param back?

Reply:
Did that, but left JSON.parse(JSON.stringify(obj)) so that the reutrn type is `object` :D